lib/ace/puppet_util.rb

```diff
@@ -2,6 +2,39 @@
 
 module ACE
   class PuppetUtil
+    CERT_DELIMITERS = /-----BEGIN CERTIFICATE-----.*?-----END CERTIFICATE-----/m
```

Comment:
.freeze to keep rubocop happy? Otherwise looks a nice little fix based on precedence.